src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -12,7 +12,7 @@ internal class AssemblyLoader
         private readonly Dictionary<string, SearchOption> dirEnumArgs;
         private readonly HashSet<AssemblyLoaderPathNameCriterion> pathNameCriteria;
         private readonly HashSet<AssemblyLoaderReflectionCriterion> reflectionCriteria;
-        private readonly TraceLogger logger;
+        private readonly LoggerImpl logger;
```

Comment:
One of the capabilities LoggerImpl provides, which maybe it shouldn't, is that it has log calls which accept the internal error code enums, for this reason, I've replaced TraceLogger with Logger only where public API surfaces are involved.  For all other internal logging purposes, we continue to use the internal logger, LoggerImpl.